### PR TITLE
Error now display correctly on Login components

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6095

## Description
The issue occurred due to incorrect jquery selectors.

## Screenshots/screencasts
https://streamable.com/wa4xvq

## Backward compatibility
This change is fully backward compatible.

## Reviewers 
@upplabs-alex-levchenko @YaroslavOvdii

### CODE REVIEW

**Comment by squallstar** [comment]
@tonytlwu can you also review this change?

File: js/build.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ Fliplet.Widget.instance('login-ds', function(data) {
           _this.removeClass('loading');
           _this.find('.btn-label').removeClass('hidden');
           _this.find('.loader').removeClass('show');
-          _this.parents('.form-btns').find('.login-error').html('Your email or password don\'t match. Please try again.').removeClass('hidden');
+          _this.find('.login-error').html('Your email or password don\'t match. Please try again.').removeClass('hidden');
```
Comment (tonytlwu) [inline js/build.js:245]: What gets shown to the user if we just use the server's error response? A `.catch()` being triggered doesn't always mean the email or password don't match. The server should be managing the scenarios (or show the JavaScript error if the error is local to the client.)

```suggestion
          _this.find('.login-error').html(Fliplet.parseError(error)).removeClass('hidden');
```

**Comment by tonytlwu** [comment]
@MaksymShokin See comment above

File: js/build.js (hunk 1 above)
Comment (MaksymShokin) [reply js/build.js]: Tested. Everything works. Fixed

**Pull request merged**